speech/test/speech_samples_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule GoogleApi.Speech.Samples.Test do
+  use ExUnit.Case
+  import ExUnit.CaptureIO
+  doctest GoogleApi.Speech.Samples
+
+  test "transcribe" do
```

Comment:
One thing that you could do is tag the tests:

```elixir
@tag :external
```

It marks the test as external so a developer can see that it's making external requests and could choose to limit which tests to run. Not a big deal because it's a single test for this entire sample.

Reply:
Also, can this test identical to the doctest?

Reply:
haha yes it is. I suppose we could only run the docs test and that would be sufficient.

Reply:
I can't figure out how to test `IO.puts` with Doctest, so I removed it.